### ISSUE TITLE
Integrate Centos CI

### ISSUE THREAD

**Issue by abompard** [fedora-infra/noggin#72]
Once the securitas project is declared in Centos CI (see [ticket 16901](https://bugs.centos.org/view.php?id=16901)), we need to make sure everything works with our `cico.pipeline` and `mergify` setup.

This is the final step of #63.

**Comment by sfinn85** [comment]
Great thanks for opening this issue @abompard on the back of another ticket outcome :). So from my understanding it is a subtask of #63 , is that right?

**Comment by sfinn85** [comment]
Hi @abompard 
Just checked out CentOS CI ticket.. it looks resolved since like 20 mins ago.. no rush, just flagging.

**Comment by abompard** [comment]
Yeah I wanted to check it's actually working before closing. Brian pinged me and Rick on IRC :-)

**Comment by sfinn85** [comment]
Perfect, when a ticket is updated in CentOS  that you logged , do you get notified through email or any other notification other then if the person pings you on IRC? Just for my own understanding of how it works

**Comment by abompard** [comment]
I did get notified by email @sfinn85 , Brian was just faster on IRC ;-)

**Comment by sfinn85** [comment]
Cool, good the email notification is there. The added step of note on IRC may be a duplicate and me pinging here is definitely a duplicate. I might suggest going forward lets let the notifications we have set up, do their thing and not add extra steps for ourselves.

**Comment by abompard** [comment]
Brian has set up our account on Centos CI, and now I need to adapt the way we run tests there because it's not doing what we need it to do. It's currently only testing on obsolete Fedora versions, and the simple macro I had set up does not allow for installation of dependencies besides tox, and we need poetry on the nodes.
So I'll rework the `.cico.pipeline` file using what has already been done in other projects, as I haven't found any documentation on its format or API.

(note: login on Centos CI is currently [broken](https://bugs.centos.org/view.php?id=16982), and that prevents me from testing my changes)

**Comment by abompard** [comment]
Login Centos CI has been fixed, but now I can't test the changes I've made to the `cico.pipeline` file because of [bug 16989](https://bugs.centos.org/view.php?id=16989). My only alternative would be to commit my tests in the master branch and I'd rather not do that, so I consider this ticket blocked.

**Comment by abompard** [comment]
I got an answer from @siddharthvipul who's trying to fix the `fedoraInfraTox()` macro and to add `poetry` to the base image that the macro uses.

**Comment by sfinn85** [comment]
Moving this to blocked as we cannot work on this until vipul gives the go ahead

**Comment by sfinn85** [comment]
Crashes in CI - team can still work on other tasks while blocked.
Hard to give timeframe when travelling ( Vipul)

**Comment by sfinn85** [comment]
Sprint 3 update: Will need to carry out test after CentOS Fix ( s )

**Comment by siddharthvipul** [comment]
@sfinn85 the repo would need to add fedora-infra-bot so that CI can pick it up
@relrod had some question regarding it's securing aspect of things, I will see if I can gather them all.
however, adding the bot would be the blocker and nothing so far on CentOS CI side

**Comment by sfinn85** [comment]
Thanks for the update @siddharthvipul.  @abompard  Can we add the repo to Fedora- infra-bot to allow CI pick this up?

**Comment by abompard** [comment]
I don't have the permissions to do that, only @relrod can do it for the moment.

**Comment by sfinn85** [comment]
Just adding notes from stand up here:

@abompard updated on stand up that he is continuing his discussion with @siddharthvipul , some bugs unblocked. 

Open question regarding repository access.

Next steps
- Get CI to work
- Discuss the possibility of adding CI to allow us to add pull request
- Discussion need to happen on the security side.

**Comment by abompard** [comment]
As a follow up I think @relrod was hesitant to add the bot to this repo's contributor's list, because it would give it write access to the code. Did I understand correctly? Are there ways to avoid that, and if not what's your current opinion on this Ricky?

**Comment by sfinn85** [comment]
Just adding notes from stand up:

Vipul working on this and will keep us posted on how things go, all going well completed today if Vipul runs into issues it could be longer.

**Comment by sfinn85** [comment]
Update from @siddharthvipul , @abompard 
"I realized the docker file needs to be merged so that I can build from them.. have meeting with Brian so that I can ask him to merge the PRs and ask him I can take a easier way to push images 😛
but yeah, so far so good, will update you after brian's meeting"

**Comment by sfinn85** [comment]
Vipul, will have an update on this tomorrow evening

**Comment by sfinn85** [comment]
Brian to merge later today if he has capacity

**Comment by siddharthvipul** [comment]
Please confirm if it works as expected :)
should be fine from CI side, if not, just ping me anywhere
